src/server.py

```diff
@@ -487,33 +476,26 @@ def run_seta_details_query():
         if head_rev or pushlog_id:
             priority = valve(head_rev, pushlog_id, branch, priority)
         else:
-            priority = None
+            priority = 0
 
     alljobs = JOBSDATA.jobtype_query()
 
     # Because we store high value jobs in seta table as default,
     # so we return low value jobs, means no failure related with this job as default
-    # when the priority is 0, otherwise we return high value jobs(1 ~ 5).
+
+    # priority = 0; run all the jobs
     if priority == 0:
+        jobtype = alljobs
+    # priority =5 run all low value jobs
+    elif priority == 5:
         low_value_jobs = [low_value_job for low_value_job in alljobs if
                           low_value_job not in jobtype]
         jobtype = low_value_jobs
+    # priority =1, run all high value jobs
+    elif priority == 1:
+        pass # use jobtype as a high value query
 
```

Comment:
so, we still make it return **high** and **low** value job? I think we are going to return a job list depend on priority that 

0 == all jobs
1 == one failure related job
2 == two failure related job 
3 == three failure related job
4 == four failure related job

which means we don't need to return low value job anymore.